# Message for sending a command down from the surface.
Header header

trajectory   desired_trajectory    # The orientation and translation we'd like to be at in this moment
io_request[] io_requests           # Any additional io requests to take place here
